tests/script_tests.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from FacialRecog import *
+from RecogScript import process_video_to_encode
+
+
+class ScriptTest(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    # all tests must start with "test"
+    def test_headless_encoding(self):
```

Comment:
![image](https://user-images.githubusercontent.com/31379789/74561607-84017780-4f2e-11ea-983e-380e86c24345.png)

not sure why but this is where the workflow failed

Reply:
I think it's because the workflow just runs the tests... Like we probably need to additionally import our image files into the workflow itself in order to get that to work :^)